src/one_or_many.rs

```diff
@@ -67,6 +67,19 @@ impl<T> OneOrMany<T> {
             }
         }
     }
+
+    pub fn to_single_mut(&mut self) -> Option<&mut T> {
```

Comment:
didnt this happen in PR #313 ?

Reply:
Yes, this is a mistake. Rebasing to fix...